tests/unit/replicator_tests.py

```diff
@@ -59,13 +59,19 @@ def tearDown(self):
         """
         Reset test attributes
         """
-        self.target_db.delete()
+        if self.client.admin_party:
+            self.target_db.delete()
+        else:
+            self.client.r_session.delete(self.target_db.database_url)
```

Comment:
I've got to be honest here.  I don't have a clue why you did this.  I assume this is what you wanted to chat about?